indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -65,6 +65,11 @@
                    :number_of_replicas 1,
                    :refresh_interval "1s"}})
 
+(def deleted-granule-setting {:index
```

Comment:
Need to set up a separate configuration for deleted granules index. Don't use the tag index shard configuration.

Reply:
Done.